tests/framework/pvc.go

```diff
@@ -62,6 +61,21 @@ func (f *Framework) CreateAndPopulateSourcePVC(pvcName string, podName string, f
 	return sourcePvc
 }
 
+// CreateAndPopulateSourceBlockPVC Creates and populates a PVC using the provided POD and command
+func (f *Framework) CreateAndPopulateSourceBlockPVC(pvcName string, podName string, fillCommand string) *k8sv1.PersistentVolumeClaim {
+	// Create the source PVC and populate it with a file, so we can verify the clone.
+	sourcePvc, err := f.CreatePVCFromDefinition(utils.NewBlockPVCDefinition(pvcName, "1G", nil, nil, "manual"))
+
+	gomega.Expect(err).ToNot(gomega.HaveOccurred())
+	// why not just call utils.CreatePod() we're not using framework to help with tests so not sure why it's even there
```

Comment:
The framework fills in the namespace and other things you would have to manually pass to utils.Whatever. that is the point of the framework.

Reply:
Copy/Paste this comment by mistake from line 55. Removed it.